kubejs/client_scripts/item_modifiers/jei_descriptions.js

```diff
@@ -191,7 +191,7 @@ onEvent('jei.information', (event) => {
             text: ['Place next to living coral to infuse.']
         },
         {
-            items: [/quark:\w+_crystal$/],
+            items: ['#quark:crystal'],
```

Comment:
Big nope, you cannot use tags in client scripts sadly :(

Reply:
seems to be working? 
![image](https://user-images.githubusercontent.com/2611674/150625329-c5bd4cb9-88d7-454f-b203-cad71bc18994.png)

Reply:
Only in singleplayer sadly, client scripts usually load before server scripts when joining a server, causing tags to be unavailable/unbound.

Reply:
Ah I see, well the regex is slightly beyond me so I'll revert for now

Reply:
It'll tag anything that begins with `quark:` and ends with `_crystal` :)

Reply:
Yeah, it just needs to also exclude the waxed variants, and myalite crystals, which currently get caught by that regex. I could just add the 9 corundum to a item list, might be the simplest option lol